src/navs/index.js

```diff
@@ -1163,7 +1163,7 @@ export const companyMenu = {
                     url: '/teams/website-vibes',
                 },
                 {
-                    name: 'Words & Pictures',
+                    name: 'Comms',
                     url: '/teams/words-and-pictures',
```

Comment:
This?

Reply:
I've learned from past experience it's easier to just leave the Strapi-controlled URLs, tbh